### ISSUE TITLE
Aggiungere la selezione della categoria alla ricerca in home page

### ISSUE THREAD

**Issue by iltempe** [emergenzeHack/covid19italia#342]
**Priorità**: bassa

## Richiesta
Una volta gestito #341 si può fare un menu a tendina in home page per vedere le categorie dove cercare.

## Risolto?

* [ ] Ho verificato che il problema è stato risolto

**Comment by cristigalas** [comment]
Concordo. Abbiamo raggiunto un numero importante di segnalazioni e la ricerca per categorie diventa indispensabile...
